src/platform/site-wide/announcements/config/index.js

```diff
@@ -59,11 +58,6 @@ const config = {
       showEverytime: true,
       relatedAnnouncements: ['welcome-to-new-va'],
     },
-    {
-      name: 'welcome-to-new-va',
-      paths: /^\/$/,
-      component: WelcomeToNewVAModal,
-    },
```

Comment:
Not super necessary but we could also remove the reference to it on line 59

Reply:
Ah yes! Great catch!